lib/huginn_acumen_product_agent/acumen_product_agent.rb

```diff
@@ -54,6 +54,10 @@ def validate_options
             unless options['contributor_types_map'].is_a?(Hash)
                 errors.add(:base, "if provided, contributor_types_map must be a hash")
             end
+
+            unless options['ignore_skus'].is_a?(Array)
+                errors.add(:base, "if provided, ignore_skus must be an array")
+            end
```

Comment:
I don't think this allows null values. It needs to.

`ignore_skus` should also default to an empty array when not provided.

Reply:
@mrogers-5s This should be fixed. Let me know what you think